File: labs/Orchestrating Containers with Docker Compose/Node/End/models/dockerCommand.js
'use strict';

var mongoose        = require('mongoose'),
    Schema          = mongoose.Schema,
    ObjectId        = Schema.Types.ObjectId;
    
var exampleSchema = Schema({
    example     : { type: String, required: true },
    description : { type: String, required: true },

});

var dockerCommandSchema = Schema({
    command                 : { type: String, required: true },
    description             : { type: String, required: true },
    examples                : [exampleSchema]
});

var DockerCommandModel = mongoose.model('dockerCommand', dockerCommandSchema);


module.exports = DockerCommandModel;

File: labs/Orchestrating Containers with Docker Compose/ASP.NETCore/Begin/AspNetCorePostgreSQLDockerApp/Models/DockerCommand.cs
using System;
using System.Collections.Generic;

namespace AspNetCorePostgreSQLDockerApp.Models {
  
  public class DockerCommand {
    public int Id { get; set; }
    public string Command { get; set; }
    public string Description { get; set; }
    public List<DockerCommandExample> Examples { get; set; }    
  }
  
}